dynamodb-client/src/main/java/org/acme/dynamodb/FruitAsyncResource.java

```diff
@@ -0,0 +1,39 @@
+package org.acme.dynamodb;
+
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("/async-fruits")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class FruitAsyncResource {
+
+    @Inject
+    FruitAsyncService service;
```

Comment:
If I'm correct, the async support was disabled for now? The quickstart should take that into account.

It should probably be stated in the guide and code should be commented here so that it works. I'm not asking to remove the code, just adjust so that the build doesn't fail.

Reply:
I was about to do so, but yesterday evening AWS SDK released a version with Netty 4.1.39. See https://github.com/quarkusio/quarkus/pull/3839 that re-enables async support.